### PR TITLE
Sof 119 generic json config

### PR DESCRIPTION
**Pull request by andrewnash**
This took me like 5 minutes, not sure if this is all we want, or if I should start looking into what's actually in the config file? Feel free to deny this request and point me in the direction of where I can learn what's in the config file. In the meantime, I just picked up another Jira.

### CODE REVIEW

**Comment by mmb186** [comment]
Awesome work for first task!

What do you think about we creating an interface for flask to use that would encapsulate the functionality to read/write to specific values. This class would be where your code could live.
@andrewnash thoughts?

**Comment by andrewnash** [comment]
@mmb186 Do we want database config dumping implemented or just ready for modification? If the ladder is true this is ready for review

File: OnLand/ControlLaptop/flaskConfig/flaskConfig.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import json
+
+# TODO: add db stuff in future?
+class FlaskConfig:
+    
+    def __init__(self, json, db):
```
Comment (mmb186) [inline OnLand/ControlLaptop/flaskConfig/flaskConfig.py]: Can we have this create the json file if it's not created to path `localStorage/` or a better name.

**Review by mmb186 — CHANGES_REQUESTED** [review]

File: OnLand/ControlLaptop/flaskConfig/flaskConfig.py (hunk 1 above)
Comment (andrewnash) [reply OnLand/ControlLaptop/flaskConfig/flaskConfig.py]: `new_config() ` creates the config file (or writes data to an existing) but In hein sight, I think it makes more sense to do this in the constructor. I will update this asap

Comment (andrewnash) [reply OnLand/ControlLaptop/flaskConfig/flaskConfig.py]: addressed in 0105bdb

**Review by mmb186 — APPROVED** [review]

**Pull request merged**